src/Microsoft.ML/Runtime/EntryPoints/MacroUtils.cs

```diff
@@ -119,7 +119,7 @@ private sealed class TaskInformationBundle
                 {
                     TrainerKinds.SignatureMultiOutputRegressorTrainer,
                     new TaskInformationBundle {
-                        TrainerFunctionName = "TrainMultiRegression",
+                        TrainerFunctionName = "MultiOutputRegressor",
```

Comment:
>MultiOutputRegressor [](start = 47, length = 20)

You said something about a name suffix... is it a concern that the suffixes are not unique? So `Regressor` above would be a suffix of any `MultiOutputRegressor`. We previously worked around this problem I guess by having every suffix start with `Train`, but that solution has gone out the window I suppose.

Reply:
Similar notes for `Classifier` vs. `BinaryClassifier`.

---
In reply to: [187424111](https://github.com/dotnet/machinelearning/pull/113#discussion_r187424111) [](ancestors = 187424111)

Reply:
OK now I know you've special cased this. :) Hmmm I'd still rather have a different solution.

---
In reply to: [187424194](https://github.com/dotnet/machinelearning/pull/113#discussion_r187424194) [](ancestors = 187424194,187424111)